docs/formats/avro.md

```diff
@@ -232,15 +234,52 @@ AS
     SELECT * from my_table;
 ```
 
+Configure the Avro encoder to send raw Avro records. The connector is associated with a SQL index
+`test_index` using the `index` property. This index specifies that the `id` field serves
+as the unique key for the view.  The connector will use this key to combine delete and insert
+operations with the same unique key into atomic updates. The encoder will generate two separate
+schemas for the key and value components of the message and publish them in the schema registry.
+
+```sql
+CREATE VIEW my_view
+WITH (
+  'connectors' = '[{
+    "index": "test_index",
```

Comment:
this is a bit redundant with the create index.
Do you expect any other purpose for create index?

Reply:
Yes, I think it would be cleaner to attach the connector directly to the index instead of attaching it to the view and referencing the index.